packages/experiments/src/components/Button/Button.test.tsx

```diff
@@ -99,4 +100,26 @@ describe('Button', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot;
   });
+
+  it('focuses correctly when focus is triggered via IButton interface', () => {
+    const wrapper = mount(
+      <div>
+        <Button content="Button 1" />
+        <Button content="Button 2" />
+        <Button content="Button 3" />
+      </div>
+    );
+
+    const buttons = wrapper.getDOMNode().querySelectorAll('button.ms-Button') as NodeListOf<HTMLButtonElement>;
+    expect(buttons.length).toEqual(3);
+
+    buttons[0].focus();
+    expect(document.activeElement!.textContent).toEqual('Button 1');
```

Comment:
You could probably assert that the DOMNode references are equal, but this is also sufficient -- https://airbnb.io/enzyme/docs/api/ShallowWrapper/equals.html. Maybe?